Observer/Order/Save.php

```diff
@@ -138,11 +139,13 @@ public function execute(Observer $observer)
                 $nostoCustomer = $this->customerRepository
                     ->getOneByQuoteId($quoteId);
                 if ($nostoCustomer instanceof NostoCustomer === false) {
-                    return;
+                    $nostoCustomerId = null;
+                } else {
```

Comment:
Remove the `else` and invert the if like this:
```
$nostoCustomerId = null;
if ($nostoCustomer instanceof NostoCustomer) {
	$nostoCustomerId = $nostoCustomer->getNostoId();
}
```
Then you can remove the `@suppress PhanTypeMismatchArgument`